xdsl/dialects/vector.py

```diff
@@ -282,6 +291,148 @@ def get(mask_operands: list[Operation | SSAValue]) -> CreateMaskOp:
         )
 
 
+def _get_mixed_position(
```

Comment:
I feel like we have a few of these around, maybe could warrant a helper to be shared across dialect? I remember being asked about the right way to do this a couple of times.